scripts/test-docs-a11y.mjs

```
@@ -0,0 +1,65 @@
+  await writeFile("test-results/docs-a11y.json", `${JSON.stringify({ tags, results }, null, 2)}\n`);
+
+  const violations = results.flatMap((result) =>
+    result.violations.map((violation) => `${result.name}: ${violation.id}`),
+  );
+  if (violations.length) throw new Error(`Axe violations found:\n${violations.join("\n")}`);
+} finally {
+  await browser?.close();
+  try {
+    process.kill(-server.pid, "SIGTERM");
+  } catch (error) {
+    if (error.code !== "ESRCH") throw error;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)